core/rfb.js

```diff
@@ -329,16 +330,33 @@
 
         // Send a key press. If 'down' is not specified then send a down key
         // followed by an up key.
-        sendKey: function (keysym, down) {
+        sendKey: function (keysym, code, down) {
```

Comment:
Indeed. If we were post-1.0 then it could have simply be handled by putting the new argument last and having some handling for it not being included.